scripts/local_testnet/setup.sh

```diff
@@ -13,11 +12,6 @@ set -o nounset -o errexit -o pipefail
 
 source ./vars.env
 
```

Comment:
```suggestion

genesis_file=${@:$OPTIND+0:1}
```

Reply:
Moved it to the other file so no longer needed.

Reply:
I’ll make the same change in siren next week!